pom.xml

```diff
@@ -86,7 +86,7 @@
         <dependency>
             <groupId>ch.cern.eam</groupId>
             <artifactId>eam-wshub-proxyclient</artifactId>
-            <version>11.6-2</version>
```

Comment:
Please remove this change from the commit, eam-wshub-core releases are done on the master branch, and might include more than one feature on each release (we need to sync other existing work)